src/Exception/ExceptionInterface.php

```diff
@@ -3,7 +3,7 @@
  * Zend Framework (http://framework.zend.com/)
  *
  * @link      http://github.com/zendframework/zf2 for the canonical source repository
- * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @copyright Copyright (c) 2005-2018 Zend Technologies USA Inc. (http://www.zend.com)
```

Comment:
As I remember we update date only when we modify something in the file. Here nothing is changed so the year shouldn't be changed.

Reply:
Where is this requirement defined?